app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -273,9 +273,17 @@ def raw_connect_v4(server, port, path, username, password, service, scheme = 'ht
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, service)
 
+      # Build the URL:
+      url = URI::Generic.build(
+        :scheme => scheme,
+        :host   => server,
+        :port   => port,
+        :path   => path
+      )
+
       # Create the connection:
       OvirtSDK4::Connection.new(
-        :url      => "#{scheme}://#{server}:#{port}#{path}",
+        :url      => url.to_s,
```

Comment:
The comment on 284 is also pretty unhelpful.  

Side question: Is OvirtSDK4::connection calling [URI.parse](https://ruby-doc.org/stdlib-2.3.3/libdoc/uri/rdoc/URI.html#method-c-parse) on this string?  If so,  it might be worthwhile supporting URI objects when creating a connection.

I'm good with this change though.